src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -250,6 +263,10 @@ def export_config(cmd,
                   # to-app-service parameters
                   appservice_account=None,
                   export_as_reference=False):
+
+    if snapshot and (key or label or skip_features or skip_keyvault):
+        raise CLIErrors.MutuallyExclusiveArgumentError("'--snapshot' cannot be specified with '--key',  '--label', 'skip-keyvault' or '--skip-features' arguments.")
```

Comment:
Same here, if possible, lets move this with the other validators. Also, there's a typo: `--skip-keyvault`

Reply:
This has been updated, thanks!